Sources/Commands/Options.swift

```diff
@@ -42,4 +42,25 @@ public class ToolOptions {
 }
 
 /// Parser conformance for ColorWrap.
-extension ColorWrap.Mode: StringEnumArgument {}
+extension ColorWrap.Mode: StringEnumArgument {
+    public static var completion: ShellCompletion {
+        return .values([
+            (Auto.description, ""),
+            (Always.description, ""),
+            (Never.description, "")
+        ])
+    }
+}
+
+/// Parser conformance for AbsolutePath.
+extension AbsolutePath: ArgumentKind {
+    public init?(arg: String) {
+        self.init(arg, relativeTo: currentWorkingDirectory)
```

Comment:
What would be a better place for this? All this is saying that `AbsolutePath` conforms to the `ArgumentKind` protocol. Incorrect usage of any API should be discouraged. I see two options going forward; 1. add a note about the API or 2. introduce an additional type that conforms to ArgumentKind, maybe named `PathArgument`.